modules/varnish/templates/default.vcl

```diff
@@ -346,6 +348,17 @@ sub vcl_hash {
 	}
 }
 
+sub vcl_hit {
+	if (!obj.ttl > 0s) {

```

Comment:
Why !obj.ttl?

Reply:
It was originally based off quite a bit of research, but now I'm not sure the vcl_hit is even necessary here as it should be purging from recv_purge anyways and called from vcl_recv so I have actually no idea why it isn't properly working anymore.